web/packages/api/src/assets_v2.ts

```diff
@@ -611,6 +695,12 @@ export async function getLocationBalance(
             }
             return BigInt(accountData?.balance ?? 0n)
         }
+        case "statemine": {
```

Comment:
You should be able to merge this case with the one above which may support PNAs in the future as well.

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/1463/commits/f309c26ea81cff0207f542a9ba8cfecc82e561de.